src/pixie_cli/install.sh

```diff
@@ -41,6 +34,17 @@ PIXIE_BANNER="
  |_|  |_|/_\_\|_|\___|
 "
 
+ARTIFACT_BASE_PATH="https://github.com/${GITHUB_REPO}/releases/download/release%2Fcli%2Fv"
+
+# Fetch latest version.
+LATEST_VERSION=""
+regex='\[CLI v(([0-9]+)\.([0-9]+)\.([0-9]+))\]'
+if [[ "$(curl -fsSL https://raw.githubusercontent.com/pixie-io/pixie/main/README.md)" =~ $regex ]]; then
+    LATEST_VERSION=${BASH_REMATCH[1]}
+fi
+
+USE_VERSION=${PX_CLI_VERSION:=${LATEST_VERSION}}
```

Comment:
We probably still want to support `PL_CLI_VERSION`. Also we previously didn't set `PL_CLI_VERSION` if it wasn't set already but this will set it (`:=` vs `:-`), I don't know if that was intentional but I don't really understand the change if it was

Reply:
Whoops `:=` was unintentional.
I have updated it now to accept both `PL_CLI_VERSION` and `PX_CLI_VERSION`, with preference towards `PX_CLI_VERSION` if both are set.